src/OrchardCore.Modules/OrchardCore.Menu/Migrations.cs

```diff
@@ -70,23 +70,6 @@ private static void MigrateMenuItems(List<ContentItem> menuItems)
         {
             foreach (var menuItem in menuItems)
             {
-                var linkMenuItemPart = menuItem.As<LinkMenuItemPart>();
-                if (linkMenuItemPart != null)
-                {
-                    // This code can be removed in a later release.
```

Comment:
There's a bunch of similar code that we can also get rid of (also here above):

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/5a4a8bc3-0d23-4a62-bc9a-2b903f226eca)